package.json

```diff
@@ -4,6 +4,7 @@
   "private": true,
   "dependencies": {
     "@chec/commerce.js": "^2.0.0-beta5",
+    "@chec/seeder": "^1.0.0-beta1",
```

Comment:
Dev only dependency maybe?

Reply:
Yeah. I wasn't sure. All the react-script stuff is a dependency... I guess you still need that to build a production bundle though...

Reply:
Our "production" for this repo is GitHub pages, which won't need this because it has a pre-defined API key right?

Reply:
I think we can sort of think of this repo as not actually having a production branch. There's really 3 repos in one here. I'll move the dependency.